build/hydrobuild.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Hydrobuild
+description: Get started with Docker Hydrobuild
+sitemap: false
+---
+
+> **Restricted**
+>
+> Docker Hydrobuild is a early-access product.
+>
+> If you want to get involved in testing Hydrobuild, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{: .restricted }
+
+Hydrobuild is a service that lets you build your container images faster,
+locally and in CI. Builds run on cloud infrastructure optimally dimensioned for
+your workloads, no configuration required. The service uses a remote build
+cache, ensuring fast builds anywhere and for all team members.
+
+## How Hydrobuild works
+
+Using Hydrobuild is no different from running a regular build. You invoke a
+build the same way you normally would, using `docker build`. The difference is
+in where and how that build gets executed.
+
+By default when you invoke a `docker build` command, your build runs on a local
+instance of BuildKit, bundled with the Docker daemon. With Hydrobuild, you send
+the build request to a BuildKit instance running remotely, in the cloud.
+
+The remote builder executes the build steps, and sends the resulting build
+output to the destination that you specify. For example, back to your local
+Docker Engine image store, or to an image registry.
+
+Using Hydrobuild comes with several benefits out of the box:
+
+- Improved build speed
+- Shared build cache
+- Native multi-platform builds
+
+And the best part: you don't need to worry about managing builders or
+infrastructure. Just connect to your builders, and start building.
+
+## Setup
+
+To get started with Hydrobuild, you need to download and install Docker
+Desktop version 4.22.0 or later.
+
+When creating a Hydrobuild builder, you specify a builder endpoint to use. An
+endpoint represents a single, isolated builder. Builder endpoints use the
+following format:
+
+```text
+cloud://<org>/<group>_<platform>
+```
+
+- `<org>` is the Docker organization that the builder is provisioned for
```

Comment:
So is this available just for Docker Business/Team customers?

Reply:
Currently only organizations can own hydrobuild builders

Reply:
Thanks for clarifying @zappy-shu. I think this should be added to the banner @dvdksn